authors/apps/authentication/tests/test_social_auth.py

```diff
@@ -1,4 +1,5 @@
 import os
+from decouple import config
```

Comment:
@BrianSerem, why did you add this line of code, it looks redundant here.

Reply:
I have removed the redundant import